paasta_tools/deployd/master.py

```diff
@@ -44,10 +46,10 @@ def process_inbox(self):
         except Empty:
             service_instance = None
         if service_instance:
-            self.log.debug("Processing {} to decide if we need to add it to bounce q".format(service_instance))
+            self.log.debug("Processing {}.{} to see if we need to add it to bounceq".format(service_instance.service,
```

Comment:
is this a typo? if not, write out 'bounce queue' so people don't ticket that you've typo'd :)